tooling/src/hypothesistooling/__main__.py

```diff
@@ -306,6 +306,29 @@ def update_python_versions():
         build_sh.chmod(0o755)
 
 
+def update_pyodide_versions():
+    vers_re = r"(\d+\.\d+\.\d+)"
+    pyodide_version = re.findall(
+        f"pyodide_build-{vers_re}-py3-none-any.whl",
+        requests.get("https://pypi.org/simple/pyodide-build/").text,
+    )[-1]
```

Comment:
Are we sure it's always sorted? Anyways:  https://pypi.org/pypi/pyodide-build/json provides the `.info.version` field

Reply:
That gives us the _most recent_ release, but we want _the largest non-prerelease version_ which means we can't use the json endpoint.  We use the regex to exclude pre-releases, but I've also switched from indexing to `max()` so we won't pick up a newer release of e.g. a patch backported to an older minor version - which hasn't happened yet and seems unlikely at zerover, but it's easy to handle 😁